schemachange/sc_records.c

```diff
@@ -572,6 +572,48 @@ static int prepare_and_verify_newdb_record(struct convert_record_data *data,
     return 0;
 }
 
+int gbl_sc_logbytes_per_second = 10000000;
```

Comment:
Consider making it an unsigned so you would not have to check for `gbl_sc_logbytes_per_second > 0` in `throttle_sc_logbytes()`.

Reply:
yes .. good idea